android/src/main/java/com/wwimmo/imageeditor/ImageEditor.java

```diff
@@ -507,6 +512,18 @@ public void save(String format, String folder, String filename, boolean transpar
 
     }
 
+    public void updateExif(String path) {
+        try {
+            File originalFile = new File(currentFilePath);
+
+            ExifInterface exif = new ExifInterface(path);
+            exif.setAttribute(ExifInterface.TAG_IMAGE_UNIQUE_ID, originalFile.getName());
```

Comment:
Would it crash if the original exif data is null?

Reply:
I don't think that the initial EXIF container can be null in the Andoroid's source code, because by default it is invited like this:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/8904023/187690856-0c8dd0e2-c31e-4d2b-833a-ad8b02ebd39a.png">

and inserted like this:
<img width="797" alt="image" src="https://user-images.githubusercontent.com/8904023/187691807-8109169f-cbdf-446b-9638-77e61f36d614.png">


So, I don't think it is possible.